### ISSUE TITLE
Wrong date/time used for graph data

### ISSUE THREAD

**Issue by stjeffrey** [rlightner/public-BrewStatus#45]
I wrote a script to import the Tilt data from an existing Google Sheet.
The data entries look correct but the graph points seem to use the date/time the data was added instead of the reading date/time.

**Comment by rlightner** [comment]
Just so I understand, you imported data into BrewStat.us from a Google Sheet via a script? Right now BrewStat.us doesn't support that functionality sadly. The date it uses is system stored when the Tilt app posts the data. No one has really requested this feature, so I don't see this feature getting any traction.

Was this a brew that you already had logging and decided to switch mid fermentation?

**Comment by stjeffrey** [comment]
That's exactly what happened.  I just got the tilt recently and with my first brew quickly realized the google graph is not the best to look at.  I just figured since the posted data has a date stamp then it would be used for plots on the graph.  So I guess the timespan field is just there to be compatible with the Tilt posted data?   I understand this being an edge case and not changing your code for it is totally fair.  It's you app and your time after all :)
